backend/tests.py

```diff
@@ -63,9 +63,81 @@ def testEnterLobby(self):
                          msg='lobby is not in waiting')
 
     def testJoinLobby(self):
+        # get parameters, test functionality
+
+
+        # get lobby and user to join
+        lobby_id = self.r.get('nextLobbyId')
```

Comment:
I dislike the naming convention here, not only is camel casing not python standard, but also things like 'Id' are unintuitive. Camelcasing an abbreviation already feels wrong no?

Reply:
We should switch to underscores but I initially wanted to stick to camel casing just because I am just personally used to them with my backend experience but I dont think its that big of a deal we should follow python standards.